Web/Assignment 2/mystyle.css

```diff
@@ -0,0 +1,57 @@
+@CHARSET "ISO-8859-1";
+body
+{
+	
+	background-color: #F0F8FF;
+	
+}
+
+.wrapper
+{
+	
+	
+	background-color: white;
+	width: 800px;
```

Comment:
ACTION: A fixed width (in pixels) should not be given to a container (until required otherwise) as it will either overflow or underflow on screens of different sizes. Divs consumes the whole width in a layout therefore width might not be required. Here in this case the container/wrapper should span to the whole width of the screen so no need to give "width"